src/Umbraco.Web/Trees/ContentTreeController.cs

```diff
@@ -36,17 +40,18 @@ public class ContentTreeController : ContentTreeControllerBase, ISearchableTree
         private readonly UmbracoTreeSearcher _treeSearcher;
         private readonly ActionCollection _actions;
 
-        public ContentTreeController(UmbracoTreeSearcher treeSearcher, ActionCollection actions)
-        {
-            _treeSearcher = treeSearcher;
-            _actions = actions;
-        }
-
         protected override int RecycleBinId => Constants.System.RecycleBinContent;
 
         protected override bool RecycleBinSmells => Services.ContentService.RecycleBinSmells();
 
         private int[] _userStartNodes;
+
+        public ContentTreeController(UmbracoTreeSearcher treeSearcher, ActionCollection actions, IGlobalSettings globalSettings, IUmbracoContextAccessor umbracoContextAccessor, ISqlContext sqlContext, ServiceContext services, AppCaches appCaches, IProfilingLogger logger, IRuntimeState runtimeState, UmbracoHelper umbracoHelper) : base(globalSettings, umbracoContextAccessor, sqlContext, services, appCaches, logger, runtimeState, umbracoHelper)
+        {
+            _treeSearcher = treeSearcher;
+            _actions = actions;
+        }
+
```

Comment:
This constructor should be moved down a little more, so it's below the `UserStartNodes` property (keeping the order of fields, properties, constructors, methods).

Reply:
That's not really a policy we enforce, I'm also not a huge fan of putting constructors below properties.

Reply:
Keeping the fields/properties together should at least be preferred, even without policies or guidelines 😉 So currently `_userStartNodes` should go up, just below `_treeSearcher` and `_actions` and the property `UserStartNodes` just above the constructor... Just to keep things tidy 😅

Reply:
aha, sorry missed that _rushing to get a PR done for hacktoberfest_ ;)